autoload/tutor.vim

```diff
@@ -180,22 +179,10 @@ function! tutor#FollowLink(force)
         elseif a:force != 1 && match(l:target, '^@tutor:') > -1
             let l:tutor = matchstr(l:target, '@tutor:\zs.*')
             exe "Tutor ".l:tutor
-        else
-            exe "help ".l:target
```

Comment:
so you've secretly been working on https://github.com/neovim/neovim/pull/1667 all this time, huh? Slow and steady wins the race I suppose...